templates/init/dataconnect/schema.gql

```diff
@@ -1,44 +1,50 @@
 # # Example schema for simple movie review app
 
-# # Users
-# # Suppose a user can leave reviews for movies
-# # user  -> reviews is a one to many relationship,
-# # movie -> reviews is a one to many relationship
-# # movie <-> user is a many to many relationship
+# # User table is keyed by Firebase Auth UID.
 # type User @table {
-#   id: String! @col(name: "user_auth")
-#   username: String! @col(name: "username", dataType: "varchar(50)")
-#   # The following are generated by the user: User! field in the Review table
-#   # reviews_on_user 
-#   # movies_via_Review
+#   # `@default(expr: "auth.uid")` sets it to Firebase Auth UID during insert and upsert.
+#   id: String! @default(expr: "auth.uid")
+#   username: String! @col(dataType: "varchar(50)")
+#   # The `user: User!` field in the Review table generates the following one-to-many query field.
+#   #  reviews_on_user: [Review!]!
+#   # The `Review` join table the following many-to-many query field.
+#   #  movies_via_Review: [Movie!]!
 # }
 
-# # Movies
+# # Movie is keyed by a randomly generated UUID.
 # type Movie @table {
-#   # The below parameter values are generated by default with @table, and can be edited manually.
-#   # implies directive `@col(name: "movie_id")`, generating a column name
```

Comment:
`@col(name: "movie_id")` is actually wrong.

The default `@col(name)` is the snake_case of the field name, so `@col(name: "id")`